pilot/pkg/features/pilot.go

```diff
@@ -657,6 +657,10 @@ var (
 	EnableLeaderElection = env.Register("ENABLE_LEADER_ELECTION", true,
 		"If enabled (default), starts a leader election client and gains leadership before executing controllers. "+
 			"If false, it assumes that only one instance of istiod is running and skips leader election.").Get()
+
+	EnableEnhancedResourceScoping = env.Register("ENABLE_ENHANCED_RESOURCE_SCOPING", false,
```

Comment:
nit: this is only describing @hzxuzhonghu 's PR; we should also add it will impact the ca root